explorer/ExplorerControls.scss

```diff
@@ -130,7 +130,7 @@ $chart-border-radius: 2px;
 .ExplorerDropdown {
     font-size: 13px;
     font-weight: 400;
-    width: 150px;
+    width: 200px;
```

Comment:
this is basically a max width. If there is not `200px` of space available, the select will shrink down to `100%` instead, and it still looks fine.
increasing the maximum available width helps by needing fewer line breaks in the options.